MSAL/src/native_auth/network/responses/validator/token/MSALNativeAuthTokenResponseValidator.swift

```diff
@@ -99,7 +99,11 @@ final class MSALNativeAuthTokenResponseValidator: MSALNativeAuthTokenResponseVal
                 .unauthorizedClient:
                 return .error(.invalidClient(message: responseError.errorDescription))
             case .invalidGrant:
-                return handleInvalidGrantErrorCodes(responseError.errorCodes, errorDescription: responseError.errorDescription, context: context)
+                if responseError.subError == .invalidOOBValue {
```

Comment:
why do you need to check the subError here? The `handleInvalidGrantErrorCodes` method is already doing that, do there is no need to add a special case for the "invalidOOBCode" error

Reply:
Actually, it is safer to check the subError instead of the error codes because in case new invalid OTP error codes are added in the future, they will be grouped under the same subError "invalid_oob_value"

Reply:
Done.